graphql/schema.graphql

```diff
@@ -369,6 +369,10 @@ type Query {
     Defines the Master Data schema to be used
     """
     schema: String
+    """
+    Defines which account will be required
```

Comment:
What does this description mean? 🤔 
Is this account an existing one in Master Data? And this parameter will be used to specify which account you're trying to get a document from?

Reply:
too late brother

Reply:
😂